packages/components/src/Banner/Banner.module.css

```diff
@@ -1,22 +1,47 @@
 .banner {
+  --banner-surface: --var(--color-surface);
+  --banner-textColor: --var(--color-text);
   display: flex;
   position: relative;
-  box-shadow: var(--shadow-base);
   padding: calc(var(--space-base) - var(--space-smaller)) var(--space-base);
-  border: var(--border-base) solid var(--color-border);
   border-radius: var(--radius-base);
-  color: var(--color-text);
-  background-color: var(--color-surface);
+  color: var(--banner-textColor);
+  background-color: var(--banner-surface);
   gap: var(--space-small);
   align-items: center;
 }
 
+.notice {
+  --banner-surface: var(--color-informative--surface);
+  --banner-textColor: var(--color-informative--onSurface);
+}
+
+.error {
+  --banner-surface: var(--color-critical--surface);
+  --banner-textColor: var(--color-critical--onSurface);
+}
+
+.warning {
+  --banner-surface: var(--color-warning--surface);
+  --banner-textColor: var(--color-warning--onSurface);
+}
+
+.success {
+  --banner-surface: var(--color-success--surface);
+  --banner-textColor: var(--color-success--onSurface);
+}
+
+.bannerChildren > a,
+.bannerChildren > p {
+  color: inherit;
+}
```

Comment:
There's some inherent risk to this selector-drilling approach so we'll want to check complex instances like the GlobalBanners in Jobber.

Reply:
I'm putting the pre-release in JO now - any specific GlobalBanners you are particularly concerned about?

Reply:
Note: addressing that brittleness [here](https://github.com/GetJobber/Jobber/pull/55959)

Reply:
@taylorvnoj I've been using a "trial" Banner as it has what I'd call the "kitchen sink" implementation

- text
- countdown
- action
- dismissible

Screenshots on the JO PR!